src/main/java/org/broadinstitute/hellbender/tools/examples/metrics/multi/ExampleMultiMetrics.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.examples.metrics.multi;
+
+import htsjdk.samtools.metrics.MetricBase;
+import org.broadinstitute.hellbender.metrics.MultiLevelMetrics;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+
+/**
+ * An example multi-level metrics collector that just counts the number of reads (per unit/level)
+ */
+public class ExampleMultiMetrics extends MultiLevelMetrics implements Serializable {
+
+    private static final long serialVersionUID = 1;
+
+    // The final metric collected for each level and for all reads.
+    public int numReads;
```

Comment:
Maybe also comment that it must be public because of the way the metrics file reflection works?